go.mod

```diff
@@ -77,3 +77,5 @@ replace (
 	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.10
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.7.0
 )
+
+replace github.com/submariner-io/admiral => github.com/skitt/admiral v0.0.0-20211215160040-faaee27f1510
```

Comment:
Is this meant for testing?

Reply:
Not just testing, it’s required because the API has changed. I’ll update this PR, but switch it to draft because I’m not happy with https://github.com/skitt/submariner-operator/pull/24 yet.